parsers/PA.py

```diff
@@ -14,158 +14,173 @@
-      'Costa Norte 3': 'gas',#[4][5]
-      'Costa Norte 4': 'gas',#[4][5]
-      'Esperanza 1': 'oil',#[7] has a single 92MW bunker fuel power plant, but [8] shows this is actually a power barge with 7 units
-      'Esperanza 2': 'oil',#[7] has a single 92MW bunker fuel power plant, but [8] shows this is actually a power barge with 7 units
-      'Esperanza 3': 'oil',#[7] has a single 92MW bunker fuel power plant, but [8] shows this is actually a power barge with 7 units
-      'Esperanza 4': 'oil',#[7] has a single 92MW bunker fuel power plant, but [8] shows this is actually a power barge with 7 units
-      'Esperanza 5': 'oil',#[7] has a single 92MW bunker fuel power plant, but [8] shows this is actually a power barge with 7 units
-      'Esperanza 6': 'oil',#[7] has a single 92MW bunker fuel power plant, but [8] shows this is actually a power barge with 7 units
-      'Esperanza 7': 'oil',#[7] has a single 92MW bunker fuel power plant, but [8] shows this is actually a power barge with 7 units
-      'Jinro': 'oil',#[6][7]
-      'Pacora 1': 'oil',#[6]
-      'Pacora 2': 'oil',#[6]
-      'Pacora 3': 'oil',#[6]
-      'PanAm 1': 'oil',#[6][7]
-      'PanAm 2': 'oil',#[6][7]
-      'PanAm 3': 'oil',#[6][7]
-      'PanAm 4': 'oil',#[6][7]
-      'PanAm 5': 'oil',#[6][7]
-      'PanAm 6': 'oil',#[6][7]
-      'PanAm 7': 'oil',#[6][7]
-      'PanAm 8': 'oil',#[6][7]
-      'PanAm 9': 'oil',#[6][7]
-      'Termocolón 1': 'oil',#[6] (spelled "Termo Colón")
-      'Termocolón 2': 'oil',#[6] (spelled "Termo Colón")
-      'Termocolón 3': 'oil',#[6] (spelled "Termo Colón")
-      'Tropitérmica 1': 'oil',#[6]:162[7] spelled "Tropitermica" in both
-      'Tropitérmica 2': 'oil',#[6]:162[7] spelled "Tropitermica" in both
-      'Tropitérmica 3': 'oil',#[6]:162[7] spelled "Tropitermica" in both
+        'ACP Miraflores 2': 'oil',  # [7] Sheet "C-GE-1A-1 CapInstXEmp"
+        'ACP Miraflores 5': 'oil',  # [7] Sheet "C-GE-1A-1 CapInstXEmp"
+        'ACP Miraflores 6': 'oil',  # [7] Sheet "C-GE-1A-1 CapInstXEmp"
+        'ACP Miraflores 7': 'oil',  # [7] Sheet "C-GE-1A-1 CapInstXEmp"
+        'ACP Miraflores 8': 'oil',  # [7] Sheet "C-GE-1A-1 CapInstXEmp"
+        'ACP Miraflores 9': 'oil',  # [7] Sheet "C-GE-1A-1 CapInstXEmp"
+        'ACP Miraflores 10': 'oil',  # [7] Sheet "C-GE-1A-1 CapInstXEmp"
+        'BLM 2': 'coal',  # [7] Sheet "C-GE-1A-2 CapInstXEmp"
+        'BLM 3': 'coal',  # [7] Sheet "C-GE-1A-2 CapInstXEmp"
+        'BLM 4': 'coal',  # [7] Sheet "C-GE-1A-2 CapInstXEmp"
+        'BLM 5': 'oil',  # [7] Sheet "C-GE-1A-2 CapInstXEmp"
+        'BLM 6': 'oil',  # [7] Sheet "C-GE-1A-2 CapInstXEmp"
+        'BLM 8': 'oil',  # [7] Sheet "C-GE-1A-2 CapInstXEmp"
+        'BLM 9': 'oil',  # [7] Sheet "C-GE-1A-2 CapInstXEmp" mentions no fuel type, and given all other units are accounted for this must be the heat recovery boiler for the 3 diesel-fired units mentioned in [2]
+        'CADASA 1': 'biomass',  # [9]
+        'CADASA 2': 'biomass',  # [9]
```

Comment:
Here is the CADASA mapping most relevant to this PR. Note that this `map_thermal_generation_unit_name_to_fuel_type` variable's intended purpose is to split the Térmicas table (i.e. thermal generation) table up into the different Electricity Map fuel types. Up until this point, that'd primarily been splitting it into oil/coal/gas by facility.

The CADASA facility is the first thermal generation fuel that maps to biomass. The operator in Panama lumps it in with thermal fuels category, rather than under Biogás (which is reasonable). However, for this reason the Electiricty Map value for `biomass` fuel will no longer simply map to the Biogás category in the _Tipos de Generación_ pie chart.